src/typeDefs/types.ts

```diff
@@ -298,6 +299,14 @@ export const types = gql`
     otpToken: String!
   }
 
+  type Venue {
+    _id: ID!
+    name: String!
+    capacity: Int!
+    description: String
+    imageUrl: URL
+  }
+
```

Comment:
Add a field known as `organization` on the here, and implement a field level resolver for the same.